package.json

```diff
@@ -38,7 +38,7 @@
     "pretest": "eslint .",
     "lint": "eslint .",
     "lint-fix": "eslint . --fix",
-    "test": "mocha --timeout 20000 --slow 0 --reporter spec test/*.spec.js"
+    "test": "mocha --timeout 20000 --slow 0 --reporter spec"
```

Comment:
please look at my comment on the original commit. This should work as it is. This helps to exclude files that are actually not tests. Like the utils